### PR TITLE
Add ledgerctl (full, legacy) + Bump Stax SDK

### PR DESCRIPTION
**Pull request by agrojean-ledger**
Most useful if we first merge : 

* https://github.com/LedgerHQ/ledgerctl/pull/73
* https://github.com/LedgerHQ/cargo-ledger/pull/31

### CODE REVIEW

File: full/Dockerfile [hunk 1]
```diff
@@ -27,6 +27,15 @@ RUN rustup target add thumbv6m-none-eabi --toolchain $RUST_NIGHTLY_VERSION
 # Adding rust-src component to nightly channel
 RUN rustup component add rust-src --toolchain $RUST_NIGHTLY_VERSION
 
+# Python packages building dependencies, can be removed afterwards
+RUN apk add -t python_build_deps python3-dev
+
+# Install ledgerctl, which is used by cargo ledger for loading or dumping an APDU app file.
+RUN pip3 install git+https://github.com/LedgerHQ/ledgerctl
```
Comment (xchapron-ledger) [inline full/Dockerfile:34]: should be on the legacy image too I guess?

File: full/Dockerfile [hunk 2]
```diff
@@ -27,6 +27,15 @@ RUN rustup target add thumbv6m-none-eabi --toolchain $RUST_NIGHTLY_VERSION
...
+
+# Cleanup, remove packages that aren't needed anymore
+RUN apk del python_build_deps
```
Comment (xchapron-ledger) [inline full/Dockerfile:37]: Does it really reduce the docker size?

Comment (agrojean-ledger) [reply full/Dockerfile:38]: I don't know haha, we could keep them... we have had a habit of removing as many pip install dependencies in other images so I figured I would do the same...

File: full/Dockerfile (hunk 1 above)
Comment (agrojean-ledger) [reply full/Dockerfile:35]: Indeed ! Since we also have cargo-ledger in legacy. I'll add it.

Comment (xchapron-ledger) [reply full/Dockerfile:35]: Could we use a tag or a specific sha1 to have a reproducible output image here?

Comment (agrojean-ledger) [reply full/Dockerfile:35]: We can definitely use at least a sha1

**Review by xchapron-ledger — COMMENTED** [review]
Can you rebase on master?

File: lite/Dockerfile [hunk 3]
```diff
@@ -86,7 +86,7 @@ RUN echo nanos2 > $NANOSP_SDK/.target
 
 # Latest Stax SDK (OS stax_1.3.0 => based on API_LEVEL 13)
 ENV STAX_SDK=/opt/stax-secure-sdk
-RUN git -C "$LEDGER_SECURE_SDK" worktree add "$STAX_SDK" v13.1.0
+RUN git -C "$LEDGER_SECURE_SDK" worktree add "$STAX_SDK" v14.1.0
```
[the remark this answers is not in the document]
Comment (agrojean-ledger) [reply lite/Dockerfile:89]: Done

**Comment by agrojean-ledger** [comment]
> name of the Pr could be changed to be more generic as it doesn't impact only the full image.

@xchapron-ledger done

**Review by xchapron-ledger — APPROVED** [review]

**Pull request merged**